src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagDispatcher.java

```diff
@@ -112,133 +93,62 @@ private static void buildEventRequestFilter() {
-            originalEventDataRequest.eventDataRequestId;
-
-        URI uri = HttpUtils.buildUri(BUGSNAG_API_URL, endpoint, null);
-
-        String response = HttpUtils.httpRequest(
-            uri,
-            CONNECTION_TIMEOUT_IN_SECONDS,
-            HttpUtils.REQUEST_METHOD.GET,
-            BUGSNAG_HEADERS,
-            null);
-
-        return JsonUtils.getPOJOFromJSON(response, BugsnagEventRequest.class);
-    }
-
-    /**
-     * Get a single organization matching the organization held in the property file.
-     */
-    private static void setOrganization() {
-
-        URI uri = HttpUtils.buildUri(BUGSNAG_API_URL, "/user/organizations/", null);
-        String response = HttpUtils.httpRequest(
-            uri,
-            CONNECTION_TIMEOUT_IN_SECONDS,
-            HttpUtils.REQUEST_METHOD.GET,
-            BUGSNAG_HEADERS,
-            null);
-
-        List<Organization> organizations = JsonUtils.getPOJOFromJSONAsList(response, Organization.class);
-
-        Organization organization = null;
-        for (Organization org : organizations) {
-            if (org.name.equalsIgnoreCase(BUGSNAG_ORGANIZATION)) {
-                organization = org;
-                break;
-            }
-        }
-
-        if (organization == null) {
-            LOG.error("Can not get required organization from Bugsnag matching organization name: "
-                + BUGSNAG_ORGANIZATION);
-        } else {
-            ORGANIZATION = organization;
-        }
-
+    public static BugsnagEventRequest newEventDataRequest() {
+        return makeEventDataRequest(null);
     }
 
     /**
-     * Get all projects configured under the provided organization id.
+     * Get a previously created event data request (if id is non-null) OR create a new request. A status of 'complete'
+     * signals that the requested data is ready to be downloaded from the populated url parameter.
+     *
+     * More here: https://bugsnagapiv2.docs.apiary.io/#reference/organizations/event-data-requests/check-the-status-of-an-event-data-request
      */
-    public static List<BugsnagProject> getProjects() {
-
-        if (ORGANIZATION == null) {
-            LOG.error("Required organization is not available. Unable to get Bugsnag projects.");
-            return null;
-        }
-
-        //TODO Consider limiting the projects to just that required by the admin dashboard
-        URI uri = HttpUtils.buildUri(
+    public static BugsnagEventRequest makeEventDataRequest(String eventDataRequestId) {
+        // Create new request if null ID is provided.
+        boolean create = eventDataRequestId == null;
+        URI eventDataRequestUri = HttpUtils.buildUri(
             BUGSNAG_API_URL,
-            "/organizations/" + ORGANIZATION.id + "/projects",
-            null);
-
-        String response = HttpUtils.httpRequest(
-            uri,
+            "organizations", BUGSNAG_ORGANIZATION, "event_data_requests", eventDataRequestId
```

Comment:
Is `BUGSNAG_ORGANIZATION` the right parameter to use here? Did the meaning of that parameter change?  Previously we would pass a Bugsnag organization ID.

Reply:
It looks like the meaning of `BUGSNAG_ORGANIZATION` has changed to be a Bugsnag organization id. README and comments in `env.yml.tmp` and `test/env.yml` should be updated accordingly.